packages/webhooks/model-validator/validator.js

```diff
@@ -14,26 +14,22 @@ class WebHookModelValidator {
     constructor (schemaPath) {
         if (typeof schemaPath !== 'string' || schemaPath.split('.').pop() !== 'graphql') {
             throw new Error('Schema file type is not supported. For now only .graphql files are supported.')
-        } else if (!fs.existsSync(schemaPath)) {
+        }
+        const resolvedPath = require.resolve(schemaPath)
+        if (!fs.existsSync(resolvedPath)) {
             throw new Error('Schema file does not exist!')
         }
-        this.schemaPath = schemaPath
+        this.schemaPath = resolvedPath
         this.models = []
         this.fieldsAJV = new Ajv()
         this.filtersAJV = new Ajv()
         this.fieldsValidators = {}
         this.filtersValidators = {}
-    }
 
-    /**
-     * Initialize validator by loading and converting GQL schema
-     * @returns {Promise<void>}
-     */
-    async init () {
-        const gqlSchema = await loadSchema(this.schemaPath, {
+        const gqlSchema = loadSchemaSync(this.schemaPath, {
             loaders: [new GraphQLFileLoader()],
         })
-        const gqlIntrospection = await introspectionFromSchema(gqlSchema)
+        const gqlIntrospection = introspectionFromSchema(gqlSchema)
```

Comment:
It looks like every model should have big object inside the memory or not?

Reply:
They have compiled AJV, which are built from minified condo shema

Reply:
As far as I know, Ajv has their own uri-system for preventing schema from being duplicated

Reply:
<img width="827" alt="image" src="https://user-images.githubusercontent.com/50069872/198971313-3ad37085-b018-45b7-9ea2-973e00addcf1.png">